cmd/alertmanager/main.go

```diff
@@ -186,6 +186,7 @@ func run() int {
 		dataDir         = kingpin.Flag("storage.path", "Base path for data storage.").Default("data/").String()
 		retention       = kingpin.Flag("data.retention", "How long to keep data for.").Default("120h").Duration()
 		alertGCInterval = kingpin.Flag("alerts.gc-interval", "Interval between alert GC.").Default("30m").Duration()
+		silenceLogging  = kingpin.Flag("silences.logging", "Enable logging silences. If it is enabled, the status change of silence will be logged").Bool()
```

Comment:
This should match the existing log-related flags like `--log.level`, so how about `"log.silences"`?

Reply:
done in #4163